### PR TITLE
Synchronize the tests

### PR DESCRIPTION
**Pull request by jlewi**
* Related to #1252

### CODE REVIEW

File: tests/stacks/ibm/test_data/expected/~g_v1_service_mysql.yaml [hunk 1]
```diff
@@ -10,6 +10,8 @@ metadata:
 spec:
   ports:
   - port: 3306
+    protocol: TCP
+    targetPort: 3306
```
Comment (animeshsingh) [inline tests/stacks/ibm/test_data/expected/~g_v1_service_mysql.yaml:14]: believe the ports are generic and independent of cloud provider - and accessed internally for mysql/minio

Comment (jlewi) [reply tests/stacks/ibm/test_data/expected/~g_v1_service_mysql.yaml:14]: @animeshsingh This is the expected output i.e. the results of running `kustomize build`. So if you think this expected output is not "correct" it indicates a bug in the kustomize manifest somewhere.

**Pull request merged**